experiments/AMIP/components/atmosphere/climaatmos.jl

```diff
@@ -1,13 +1,11 @@
-# atmos_init: for ClimaAtmos pre-AMIP interface
-using StaticArrays
-using Statistics: mean
-using LinearAlgebra: norm
+# atmos_init: for ClimaAtmos interface
+import StaticArrays as SA
+import Statistics as Stats
+import LinearAlgebra as LinAlg
```

Comment:
I wonder if we could use full names for these actually, since we're not using them very often, and use shortcuts for the clima packages (you're already doing this for most of it but maybe we can use CC for ClimaCore as well.

Reply:
Sounds good, thank you Lenka!